packages/expo-cellular/ios/EXCellular/CellularModule.swift

```diff
@@ -56,6 +56,21 @@ public class CellularModule: Module {
       return .unknown
     }
   }
+  
+  
+  static func getCurrentCellularInfoAsync() -> [String: Any?] {
```

Comment:
It's ok to put this function here, but it doesn't "export" itself to JS. You need to add a `method` in the definition at the top of the class. Also, you can change `constants` to return the same thing